templates/Program/program_description.html.twig

```diff
@@ -26,10 +30,60 @@
           </span>
```

Comment:
After running the code, i found that the disabled input doesn't look obvious at all. There is no visual indication. So can we add a spinner inside the text box, and when the api is loading, in addition to disabling the input, also show the spinner and also clear the input text `editDescription.val('')`

Patch below that you can directly paste in intellij Git->Patch -> apply patch from clipboard

```
Index: templates/Program/program_description.html.twig
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/templates/Program/program_description.html.twig b/templates/Program/program_description.html.twig
--- a/templates/Program/program_description.html.twig	(revision 437f010edea44f9a66f8487cbca15869eb831c9e)
+++ b/templates/Program/program_description.html.twig	(date 1641500418840)
@@ -22,6 +22,11 @@
             class="mdc-text-field mdc-text-field--filled mdc-text-field--textarea mdc-text-field--no-label description">
           <span class="mdc-text-field__ripple"></span>
           <span class="mdc-text-field__resizer">
+              <span id="editor_loading_spinner">
+                <i class="ms-2 material-icons">
+                  {% include 'components/loading_spinner.html.twig' with {'size': 'small'} %}
+                </i>
+              </span>
               <textarea class="mdc-text-field__input mt-2 mb-2" id="edit-description" name="edit-description"
                         title="_edit-description"
                         maxlength="{{ max_description_size }}" style="width: 100%;" rows="10" cols="100">
```

Reply:
I've added this in for the description and credit. It looks good 👍. Good idea